src/components/vendors/VendorRfpManager.tsx

```
@@ -0,0 +1,612 @@
+              const isSelected = rfp.id === selectedRfpId;
+              const bidsCount = (bidsByRfp[rfp.id] || []).length;
+
+              return (
+                <div
+                  key={rfp.id}
+                  onClick={() => setSelectedRfpId(rfp.id)}
+                  className={cn(
+                    "p-3.5 border-2 rounded-lg cursor-pointer transition-all space-y-1.5",
+                    isSelected
+                      ? "border-black bg-white shadow-[2px_2px_0px_0px_rgba(0,0,0,1)] ring-1 ring-emerald-400"
+                      : "border-gray-300 bg-gray-50/70 hover:border-gray-500"
```

ESLint

[error] Insert `,`

(prettier/prettier)